src/constants.ts

```diff
@@ -22,12 +22,14 @@ export const nativeFeaturesFromVersion: NativeFeaturesFromVersion = {
             fromVersion: '11.71.0',
         },
         geolocation: { fromVersion: '11.71.0' },
+        saveBackStack: { fromVersion: '12.29.03' },
```

Comment:
```suggestion
        safeBackStack: { fromVersion: '12.29.03' },
```

save – это глагол) safe – прилагательное